workflows/run_dev_se.nf

```diff
@@ -0,0 +1,76 @@
+/***********************************************************************************************
+| WORKFLOW: PREPROCESSING ON SHORT-READ MGS DATA (EITHER SINGLE-END OR PAIRED-END) |
+***********************************************************************************************/
+
+import groovy.json.JsonOutput
+import java.time.LocalDateTime
+
+/***************************
+| MODULES AND SUBWORKFLOWS |
+***************************/
+
+include { RAW } from "../subworkflows/local/raw"
+include { CLEAN } from "../subworkflows/local/clean"
+include { PROCESS_OUTPUT } from "../subworkflows/local/processOutput"
+include { LOAD_SAMPLESHET } from "../subworkflows/local/loadSampleSheet"
+nextflow.preview.output = true
+
+/*****************
+| MAIN WORKFLOWS |
+*****************/
+
+// Complete primary workflow
+workflow RUN_DEV_SE {
+    // Start time
+    start_time = new Date()
+    start_time_str = start_time.format("YYYY-MM-dd HH:mm:ss z (Z)")
+
+    // Check if grouping column exists in samplesheet
```

Comment:
I feel like this should be in LOAD_SAMPLESHEET

Reply:
Done